test/serverModel-test.ts

```diff
@@ -33,6 +33,10 @@ describe('Sever Model Utility', () => {
         severity: 0,
         trace: 'trace'
     };
+    const createStatus: Protocol.CreateServerResponse = {
+        status: status,
+        invalidKeys: []
```

Comment:
Any reason why it's not typed ?

Reply:
What exactly is not typed here?

Reply:
The invalidKeys array

Reply:
invalidKeys type is inferred from here:
https://github.com/redhat-developer/rsp-client/blob/master/src/protocol/protocol.ts#L29

This is just an assignment of empty array into a string array.

Reply:
@jrichter1 so why not type it here, too?

Reply:
because this is an assignment into an already typed variable